ziti/cmd/ziti/cmd/config_templates/controller.yml

```diff
@@ -1,83 +1,37 @@
 v: 3
 
 #trace:
-#  path:                 ctrl.trace
+#  path: "{{ .ZitiCtrlRawname }}.trace"
 
-profile:
-#   cpu:
-#       path: /home/plorenz/tmp/ctrl.cpu.pprof
+#profile:
 #  memory:
 #    path: ctrl.memprof
 
-db: {{.DatabaseFile}}
+db:                     "{{ .ZitiHome }}/db/ctrl.db"
```

Comment:
i kinda hate to point this out - but on windows this will end up being "ugly". example:
```
  cert:                 "C:\Users\cd\.ziti\quickstart\sg3\pki/sg3-intermediate/certs/MyRouterName-client.cert"
  server_cert:          "C:\Users\cd\.ziti\quickstart\sg3\pki/sg3-intermediate/certs/MyRouterName-server.cert"
```
is it easy enough to replace all \ with / and at least keep the directory separators consistent?

Reply:
Done